fe/fe-core/src/main/java/org/apache/doris/task/AgentBatchTask.java

```diff
@@ -172,7 +173,6 @@ public void run() {
                     try {
                         agentTaskRequests.add(toAgentTaskRequest(task));
                     } catch (Exception e) {
-                        task.failed();
```

Comment:
delete  "try catch" at line 172.  it only throw, no really  handle the exception